network/vpc/troubleshooting/vpc-limitations.mdx

```diff
@@ -0,0 +1,28 @@
+---
+meta:
+  title: VPC Limitations
+  description: This page sets out some known limitations of Scaleway's VPC
+content:
+  h1: VPC Limitations
+  paragraph: This page sets out some known limitations of Scaleway's VPC
+tags: vpc regional-private-networks
+dates:
+  validation: 2023-03-22
+  posted: 2023-03-22
+categories:
+  - network
+---
+
+This page sets out some current limitations of Scaleway's VPC.
+
+- VPC is currently only available in the Paris region.
```

Comment:
Add that at the moment there is only one VPC at the region (The "Default VPC") per project.
It is not possible to create, update or delete it